sdv/logging/utils.py

```diff
@@ -0,0 +1,91 @@
+"""Utilities for configuring logging within the SDV library."""
+
+import contextlib
+import logging
+import logging.config
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
+
+
+def get_sdv_logger_config():
+    """Return a dictionary with the logging configuration."""
+    logging_path = Path(__file__).parent
+    with open(logging_path / 'sdv_logger_config.yml', 'r') as f:
+        logger_conf = yaml.safe_load(f)
+
+    # Logfile to be in this same directory
+    for logger in logger_conf.get('loggers', {}).values():
+        handler = logger.get('handlers', {})
+        if handler.get('filename') == 'sdv_logs.log':
+            handler['filename'] = logging_path / handler['filename']
```

Comment:
you had ideas for an alternate place to store right? @pvk-developer

Reply:
Yes, my idea was to add `platformdirs` and then store it locally there (it would be similar to where we stored credentials on `SDV-Installer`).
```python
    store_path = user_app_path = Path(platformdirs.user_data_dir('sdv', 'sdv-dev'))
```

Reply:
I think this would be fine